pkg/storage/gcp/lister.go

```diff
@@ -2,11 +2,33 @@ package gcp
 
 import (
 	"context"
+	"strings"
+
+	"github.com/gomods/athens/pkg/storage"
 )
 
 // List implements the (./pkg/storage).Lister interface
 // It returns a list of versions, if any, for a given module
 func (s *Storage) List(module string) ([]string, error) {
 	ctx := context.Background()
-	return s.bucket.ListVersions(ctx, module)
+	unfiltered, err := s.bucket.List(ctx, module)
+	if err != nil {
+		return nil, err
+	}
+	versions := make([]string, 0, 10)
+	for _, n := range unfiltered {
+		// kinda hacky looking at this time
```

Comment:
if you have any ideas for making this less hacky, can you create an issue for it @marpio ?

Reply:
that was my comment from another PR :) but I haven't thought about it yet. It might in fact just appear hacky.

Reply:
I actually think this is fine. I'll extract a func.